src/certificateForm/CertificateFormPage.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { H2 } from '../commons/headings'
+import FormCertificate from './FormCertificate'
+
+export default function ChartPage({ title, categoryList, onProgressChange }) {
```

Comment:
Right now it looks like you are not using the props categoryList and onProgressChange. If it is link that, you don't need to destructure them.

Reply:
ups, copy-paste error ;-)